test/extensions/filters/network/common/fuzz/BUILD

```diff
@@ -36,6 +36,19 @@ envoy_proto_library(
     ],
 )
 
+_readfilter_fuzz_filters = [
```

Comment:
Please add a small comment describing this field (motivation for its existence)

s/_readfilter_fuzz_filters/READFILTER_FUZZ_FILTERS/

Reply:
Done